.github/workflows/build-and-deploy.yml

```
@@ -12,6 +12,9 @@ on:
     branches:
       - "main"
       - "release-*"
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
```

why ignore only these two type of files and why all yml file in all subpaths?

i suggest to use an "allowed-list" strategy, so:
```suggestion
    paths:
      - "**/src/main/**"
      - "**pom.xml"
      - "Dockerfile"
      - ".github/workflows/build-and-deploy.yml"
```

---

fixed with https://github.com/pagopa/pdv-ms-person/pull/47/commits/2841ba68e2a42980577cf11d77944c9bc3dfae57